src/instance/minecraft/CommandsManager.ts

```diff
@@ -76,6 +75,20 @@ export class CommandsManager extends EventHandler<MinecraftInstance> {
       return true
     }
 
+    if (commandName === 'help' || commandName === 'command') {
+      if (arguments_.length <= 0) {
+        await minecraftInstance.send(`/gc Commands: ${this.commands.map((command) => command.name).join(', ')}`)
+        return false
```

Comment:
Must return `true`. `true` means a command has been processed/detected. It is a flag for the parent executor, used to take into account when displaying the message later.

Reply:
oops, ill change it now